README-CN.md

```diff
@@ -1,65 +1,127 @@
+```
 
-### 使用 YAML 部署 ob-operator
+通过以下命令验证 ob-operator 部署成功：
 
-1. 部署 CRD 相关文件。
+```shell
+kubectl get pod -n oceanbase-system
 
-```yaml
-kubectl apply -f ./deploy/crd.yaml
+# desired output 
+NAME                                            READY   STATUS    RESTARTS   AGE
+oceanbase-controller-manager-86cfc8f7bf-4hfnj   2/2     Running   0          1m
 ```
 
-2. 部署 ob-operator 相关文件。
+### 部署 OceanBase 集群
+
+创建 OceanBase 集群之前，需要先创建好若干 secret 来存储 OceanBase 中的特定用户的密码：
+
+```shell
+kubectl create secret generic root-password --from-literal=password='root_password'
+kubectl create secret generic proxyro-password --from-literal=password='proxyro_password'
+kubectl create secret generic monitor-password --from-literal=password='monitor_password'
+kubectl create secret generic operator-password --from-literal=password='operator_password'
+```
 
-注意配置 `--cluster-name` 启动参数，推荐与 Kubernetes 集群名称保持一致。
+通过以下命令即可在 K8s 集群中部署 OceanBase：
 
-```yaml
-kubectl apply -f ./deploy/operator.yaml
+```shell
+kubectl apply -f https://raw.githubusercontent.com/oceanbase/ob-operator/2.1.0_release/example/quickstart/obcluster.yaml
 ```
 
-3. 配置节点 label。
+一般初始化集群需要 2 分钟左右的时间，执行以下命令，查询集群状态，当集群状态变成 running 之后表示集群创建和初始化成功：
+
+```shell
+kubectl get obclusters.oceanbase.oceanbase.com test
+
+# desired output 
+NAME   STATUS    AGE
+test   running   6m2s
+```
 
-需要将 Kubernetes 节点打 label，label 需要与 obcluster.yaml 中 `nodeSelector` 配置相匹配。
-ob-operator 会将 Pod 调度到具有相应 label 的节点上。
+### 连接集群
 
-推荐配置 label 的 key 为 `topology.kubernetes.io/zone`。
+通过以下命令查找 observer 的 POD IP，POD 名的规则是 {cluster_name}-{cluster_id}-{zone}-uuid：
 
-```yaml
-kubectl label node nodename topology.kubernetes.io/zone=zonename
+```shell
+kubectl get pods  -o wide
 ```
 
-4. 部署 OceanBase 集群。
+通过以下命令连接：
 
-```yaml
-kubectl apply -f ./deploy/obcluster.yaml
+```shell
+mysql -h{POD_IP} -P2881 -uroot -proot_password oceanbase -A -c
 ```
 
-5. 集群部署成功后可以使用 Service 的 ClusterIP 连接 OceanBase 集群。
+## 项目架构
+
+ob-operator 以 kubebuilder 为基础，通过统一的资源管理器接口、全局的任务管理器实例以及解决长调度的任务流机制完成对 OceanBase 集群及相关应用的控制和管理。ob-operator 的架构大致如下图所示：
+
+![ob-operator 架构设计](./docs/img/ob-operator-arch.png)
+
+有关架构细节可参见[架构设计文档](./docs/zh_CN/arch.md)。
+
+## 特性
+
+ob-operator 支持 OceanBase 集群的管理、租户管理、备份恢复、故障恢复等功能，具体而言支持了以下功能：
+
+- [x] 集群管理：集群自举、调整集群拓扑、支持 K8s 拓扑配置、扩缩容、集群升级、修改参数
+- [x] 租户管理：创建租户、调整租户拓扑、管理资源单元、修改用户密码
+- [x] 备份恢复：向 OSS 或 NFS 目的地周期性备份数据、从 OSS 或 NFS 中恢复数据
+- [x] 物理备库：从备份中恢复出备租户、创建空备租户、备租户升主、主备切换
+- [x] 故障恢复：单节点故障恢复，IP 保持情况下的集群故障恢复
+
+即将支持的功能有：
+
+- [ ] Dashboard：基于 ob-operator 的图形化 OceanBase 集群管理工具
+- [ ] 丰富的运维任务资源：包括但不限于针对集群和租户的轻量任务
+
+
+## 支持的 OceanBase 版本
+
+支持 OceanBase v4.x 版本，已经验证过的有 4.1.x 和 4.2.x 版本，后续会不断支持 OceanBase 社区版的新版本。
+
+暂不支持 OceanBase v3.x 版本。
+
+## 环境依赖
+
+ob-operator 使用 [kubebuilder](https://book.kubebuilder.io/introduction) 项目进行构建，所以开发和运行环境与其相近。
+
+* 构建 ob-operator 需要 Go 1.20 版本及以上；
+* 运行 ob-operator 需要 Kubernetes 集群和 kubectl 的版本在 1.11.3 及以上；
```

Comment:
1.11.3 is such an old version, we haven't tested on that

Reply:
This is the requirements of kubebuilder@v3, so I take it.

Reply:
I update it to 1.18, and demonstrate what versions we have tested.